core/bounds.gms

```diff
@@ -217,7 +217,9 @@ if (cm_nucscen eq 5,
 );
 
 *** -------------------------------------------------------------
-*** *DK* Phaseout of 1st generation biofuel technologies
+*** *DK* Force no new capacities of 1st generation biofuel technologies to be
```

Comment:
if we delete *DK*, I think this could go into iur in code docu, right?

Reply:
yes, should I delete "*DK*"? I think it meant David wrote this?

Reply:
In theory the programmer responsible for the code should be tracked by git blame now, but the old signatures, I am not sure if we should keep them (they should in theory be traceable to the old version control system)